x-pack/legacy/plugins/ml/public/datavisualizer/index_based/breadcrumbs.ts

```diff
@@ -4,11 +4,14 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-// @ts-ignore
-import { ML_BREADCRUMB } from '../../breadcrumbs';
+import {
+  ML_BREADCRUMB,
+  DATA_VISUALIZER_BREADCRUMB,
+  // @ts-ignore
+} from '../../breadcrumbs';
 
 export function getDataVisualizerBreadcrumbs() {
```

Comment:
For the main page, is is possible for the breadcrumbs to be `Machine Learning / Data Visualizer / Index` here:

![image](https://user-images.githubusercontent.com/7405507/64716735-d390d280-d4ba-11e9-8f8c-d18604b7f627.png)